src/configs/winston.ts

```diff
@@ -0,0 +1,43 @@
+import * as winston from 'winston';
+import os from 'os';
+
+const homedir = os.homedir();
+const logsPath = `${homedir}/.express/logs/reveal-express-server.log`
```

Comment:
Can we make this configurable please

Reply:
Sure.
Was looking at the possibility of using the default logs folder `/var/log/` but am experiencing some problems with write permissions.